### DISCUSSION TITLE
Merge Queues are getting stuck and not running the required checks (again!!)

### DISCUSSION THREAD

**Discussion by azarudeena** [community/community#28372] [Pull Requests]
This is great feature and productivity hack for the team. but somehow, this is getting impacted often. pains of being in beta world. 

![image](https://user-images.githubusercontent.com/53475043/182917935-87e87034-4826-4d59-b2aa-0280023fb163.png)

**Comment by dhess** [comment]
This now appears to be working for us, after being stuck for at least 6 hours. 

I'm pretty disappointed thus far in this feature: it's fairly unreliable, has practically no controls or useful UI, and has not appeared to make any significant progress, either in terms of functionality, controls, UI, or reliability, in the several months we've been using it.
